spec-0008/index.md

```diff
@@ -0,0 +1,227 @@
+# SPEC-8: Securing the Release Process
+
+---
+
+title: "SPEC 8 — Securing the Release Process"
+number: 8
+date: 2024-06-04
+author:
+
+- "Matthew Feickert <matthew.feickert@cern.ch>"
+- "Pamphile Roy <roy.pamphile@gmail.com>"
+- "Juanita Gomez <juanitagomezr2112@gmail.com>"
+- "Seth Larson <sethmichaellarson@gmail.com>"
+- "Lars Grüter <lagru@mailbox.org>"
+  discussion: https://discuss.scientific-python.org/t/spec-8-supply-chain-security
+  endorsed-by:
+
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the recommendation.
+https://github.com/scientific-python/summit-2024/issues/9
+-->
+
+Securing the Open Source supply chain (OSSC) is becoming a more relevant concern in recent years, with examples of sophisticated attacks against the ecosystem (e.g., the 2024 [`xz` utils backdoor](https://en.wikipedia.org/wiki/XZ_Utils_backdoor)) and [malware attacks on PyPI](https://blog.pypi.org/posts/2024-04-10-domain-abuse/) highlighting the need for supply chain security to be taken seriously.
```

Comment:
Maybe say why it is a more relevant concern?

Something along the lines of: Open source libraries are now commonplace in many research and industry software stacks. Securing the OSSC is therefore an increasing concern, with examples of ...